$NetBSD: patch-configure,v 1.24 2025/04/03 16:33:06 tsutsui Exp $

- setuid root / setgid utmp is handled by SPECIAL_PERMS in pkgsrc

--- configure.orig	2025-03-29 23:13:10.000000000 +0000
+++ configure
@@ -28392,16 +28392,16 @@ if test "$INSTALL_OPT" = ""; then
 	if test "$utmp_suid" = "yes" -a "$UTMP_NAME" != "none" -a "$gui" != "quartz"; then
 		has_utmp=`grep utmp /etc/group 2>/dev/null`
 		if test "$has_utmp" ; then
-			INSTALL_OPT="-m 2755 -g utmp"
+			INSTALL_OPT=""
 		else
-			INSTALL_OPT="-m 4755 -o root"
+			INSTALL_OPT=""
 		fi
 	else
-		INSTALL_OPT="-m 755"
+		INSTALL_OPT=""
 	fi
 
 	if test "$gui" = "fb" ; then
-		INSTALL_OPT="-m 4755 -o root"
+		INSTALL_OPT=""
 	fi
 fi
 
